### DISCUSSION TITLE
Disable middle click paste.

### DISCUSSION THREAD

**Discussion by Muizzyranking** [hyprwm/Hyprland#6934] [Q&A]
Recently installed hyprland and I have 3 fingers swipe to switch workspace. Sometimes when  I am typing and i want to swith workspace, i mistakenly paste. I want to disable this paste totally. I saw this in the wiki [](https://wiki.hyprland.org/Configuring/Variables/#misc)  to disable middle click paste 
```
middle_click_paste	whether to enable middle-click-paste (aka primary selection)	bool	true

```
which i believeed should work and then i get an error that `misc:middle_click_paste does not exist` and i think it is because i do not have the latest version of hyprland installed becuase I am using fedora. What was the old option for that?

**Comment by fufexan** [comment]
`bindn = , mouse:274, exec, wl-copy -pc` might work (https://github.com/hyprwm/hyprland-wiki/pull/571/files)
